### ISSUE TITLE
Monument Cards

### ISSUE THREAD

**Issue by KasiKarra** [thyer/CS340winter2016#669]
Monument cards can't be used to win and also can't be used ever.

**Comment by cmriboldi** [comment]
That's not true. They can only be used to win. And they can't be used on other turns. That's part of the rules and built into our game. You can only play a monument card if you have enough monument cards to win the game.

**Comment by thyer** [comment]
Agree with @cmriboldi unless that wasn't the intent of the bug being opened.

**Comment by KasiKarra** [comment]
I had 9 points and 1 monument, but I wasn't able to use the monument.  Therefore I mentioned that monuments can't be used ever.  I didn't know if this picture was very good so I didn't post it.  Sorry

![monument](https://cloud.githubusercontent.com/assets/16614325/14083738/7e28de9c-f4d4-11e5-95b4-618386409672.png)

**Comment by cmriboldi** [comment]
This picture shows that you have 8 points. If you have 8 points and only one monument then you shouldn't be able to play a monument.

**Comment by thyer** [comment]
It looks as though the TurnTracker and ResourceBar actually disagree here.
